headers/functions/overlay29.h

```diff
@@ -105,6 +105,7 @@ void TryPointCameraToMonster(struct entity* entity, undefined param_2, undefined
 void RestorePpAllMovesSetFlags(struct entity* entity);
 bool ShouldMonsterHeadToStairs(struct entity* entity);
 bool MewSpawnCheck(enum monster_id monster_id, bool fail_if_mew);
+bool TeamMemberHasExclusiveItemEffectActive(enum exclusive_item_effect_id effect_id);
```

Comment:
Reorder to match the symbol file, should be between `ExclusiveItemDefenseBoost` and `TrySpawnEnemyItemDrop`